qa/workunits/ceph-helpers.sh

```diff
@@ -197,27 +223,17 @@ function kill_daemons() {
     local dir=$1
     local signal=${2:-TERM}
     local name_prefix=$3 # optional, osd, mon, osd.1
-    local delays=${4:-0 0 1 1 1 2 3 5 5 5 10 10 20 60 60 60 120}
-
+    local delays=$4 #optional timing
```

Comment:
It's not used for anything but testing the function itself and such testing should now be done in kill_daemon.